src/System.Diagnostics.DiagnosticSource/src/System.Diagnostics.DiagnosticSource.csproj

```diff
@@ -62,7 +56,7 @@
   <ItemGroup Condition=" '$(TargetGroup)' == 'netcoreapp' OR '$(TargetGroup)' == 'uap' OR '$(TargetGroup)' == 'netstandard1.1' OR '$(TargetGroup)' == 'netstandard1.3' OR '$(TargetGroup)' == 'netstandard'">
     <Compile Include="System\Diagnostics\Activity.DateTime.corefx.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'net46' OR '$(TargetGroup)' == 'netfx'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'net45' OR '$(TargetGroup)' == 'net46' OR '$(TargetGroup)' == 'netfx'">
```

Comment:
Did you test building a NuGet package with these changes and does it indeed make it work on .NET Framework 4.5.* ?

Because, as far as I understand, the only .NET Framework TargetGroup that is being built anymore is 'netfx'.  The use of 'net45' or 'net46' is meaningless now.

cc: @weshaggard @ericstj